app/views/blacklight/nav/_bookmark.html.erb

```diff
@@ -1,2 +1,3 @@
 <%= link_to t('blacklight.header_links.bookmarks'), "/bookmarks", class: 'dropdown-item' %>
 <%= link_to t('blacklight.header_links.bookmarks_fa'), bookmarks_path, class: 'dropdown-item' %>
+<span class="badge badge-secondary bg-secondary" style="display: none" data-role='bookmark-counter'><%= current_or_guest_user.bookmarks.count %></span>
```

Comment:
Use Bootstrap's visiblity utilities instead of inline styles: https://getbootstrap.com/docs/5.3/utilities/visibility/

Reply:
this leads to weird formatting, where the menu has a blank entry now. From the bootstrap page ".invisible elements still take up space in the page." - this is likely the cause of the blank entry. 

using .invisible class
![image](https://github.com/nla/nla-arclight/assets/153031672/fb1e2c09-3ca0-47e8-8eef-d8ca7f4ce3fe)

using inline css
![image](https://github.com/nla/nla-arclight/assets/153031672/5938c14f-9ebd-41fb-9d56-286ff910ed1f)

Reply:
Try https://getbootstrap.com/docs/5.3/helpers/visually-hidden/. There used to be just one way to do this in Bootstrap 4.

Reply:
visually-hidden works for this. thanks!